resident-services/scripts/Resident_Test_script.jmx

```diff
@@ -4464,6 +4464,774 @@ vars.put(&quot;utcTime&quot;, utcTime);</stringProp>
           <hashTree/>
         </hashTree>
       </hashTree>
+      <ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Track Service Request (Preparation)" enabled="true">
+        <stringProp name="ThreadGroup.on_sample_error">continue</stringProp>
+        <elementProp name="ThreadGroup.main_controller" elementType="LoopController" guiclass="LoopControlPanel" testclass="LoopController" testname="Loop Controller" enabled="true">
+          <boolProp name="LoopController.continue_forever">false</boolProp>
```

Comment:
Channel Verification Status (Preparation) - since loop controller is used, are we reusing the same transaction id for both email and phone ?

Reply:
No, we are not using same transaction id for both email and phone but using same individual id for both.